src/components/dashboard/FaceVerification/api/FaceVerificationApi.js

```diff
@@ -43,7 +44,7 @@ class FaceVerificationApi {
     } catch (exception) {
       const { message } = exception
 
-      logger.error('Session session token issue failed:', message, exception)
+      logIssue(logger, 'Session token issue failed:', message, exception)
```

Comment:
you could use a more explicit way of logging like you did in GoodServer
shouldLogError() ? log.error('msg') : log.warn('msg')

Reply:
yes please

Reply:
done